crt_portal/static/js/data-filters.js

```diff
@@ -88,8 +88,37 @@
     document.querySelector('.data-content').hidden = false;
   }
 
+  function initCityFilter(filter) {
+    filter?.addEventListener('change', () => {
+      filterByCity(filter.value);
```

Comment:
WDYT (might be a good candidate for a stacked or followup PR even) about making these shareable via setParam() / getParam()